### ISSUE TITLE
[SI-ERROR-TRUE] 3.2.2 On Input - Missing button in form

### ISSUE THREAD

**Issue by cathysnider** [CuBoulder/express#978]
## Problem Description per Siteimprove

The page contains a form but there is no submit button. The form should be provided with a submit button to maximize predictability.

## Page Description

- An Article List Page containing all Articles created; no filtering and no exposed filters.
- A People List Page containing all Persons created; no filtering and no exposed filters.

## What we're doing

There's a form tag directly below the title and above the articles. The form does not show on the page, though it is not completely empty - it has three hidden input tags:
< form action="/p1d5347e3518/news" method="post" id="cu-article-list-filter-form" accept-charset="UTF-8" style="z-index: 2147483646; position: relative;" >
< input type="hidden" name="form_build_id" value="form-dLaGh" >
< input type="hidden" name="form_id" value="cu_article_list_filter_form" >

## Solution

none suggested
